contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -0,0 +1,289 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import {RouterBase, ITypeAndVersion} from "./RouterBase.sol";
+import {IFunctionsRouter} from "./interfaces/IFunctionsRouter.sol";
+import {IFunctionsCoordinator} from "./interfaces/IFunctionsCoordinator.sol";
+import {AuthorizedOriginReceiver} from "./accessControl/AuthorizedOriginReceiver.sol";
+import {IFunctionsSubscriptions, FunctionsSubscriptions, IFunctionsBilling} from "./FunctionsSubscriptions.sol";
+
+contract FunctionsRouter is RouterBase, IFunctionsRouter, AuthorizedOriginReceiver, FunctionsSubscriptions {
+  event RequestStart(bytes32 indexed requestId, Request request);
+  event RequestEnd(
+    bytes32 indexed requestId,
+    uint64 indexed subscriptionId,
+    uint96 totalCostJuels,
+    address transmitter,
+    FulfillResult resultCode,
```

Comment:
It would because it is the Router's interface. I think this is okay, because the Router won't be able to throw new codes anyway. If a new code would originate from what we _can_ update (the Coordinator) we can handle that there.

Alternatively we can keep the enum definition in a different place and just return `int`s

Reply:
Made the change using comments to make it readable. Check it out!